build/pipelines/templates/release-sign-and-bundle.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     - Buildx64Release
     - Buildx86Release
     - Buildarm64Release
+    - CodeFormatCheck
   condition: |
     and
     (

```

Comment:
technically y'gotta add "CodeFormatCheck" to the condition here to make sure it _actually succeeds_ before proceeding with the bundling... maybe